website/docs/advanced-concepts/version-control-with-git/guides/resolve-merge-conflicts-in-git.md

```diff
@@ -0,0 +1,123 @@
+# Resolve Merge Conflicts in Git
+
+Merge conflicts are a regular part of working with Git, arising when changes from separate branches cannot be merged. 
+
+This guide provides strategies for resolving different types of merge conflicts, ensuring smoother collaboration and project management in Git.
+
+
+
+### Resolve via PR Interface
+Resolving conflicts via the Git PR interface is recommended when conflicts involve a small number of files with straightforward changes.
+
+1. Once you have committed the changes to a particular branch, go to your Git provider and raise a Pull Request. For example, if you encounter merge conflicts while merging branch `b1` into `b2`, raise a PR.
+
+2. Scroll down to the bottom of the PR page and click the **Resolve conflicts** button. If the resolve button is disabled, you need to resolve using the command-line interface to resolve conflicts.
+
+<dd>
+
+<ZoomImage src="/img/conflicts-git-ui.png" alt="" caption=""/>
+
+</dd>
+
+3. You can see a list of files and pages with merge conflicts. Select a conflicted file/page to begin resolving conflicts.
+
+
+4. In the editor, you can see the lines where the conflicts have occurred:
+
+<dd>
+
+The conflict markers used by Git are `<<<<<<<`, `=======`, and `>>>>>>>.` The code between `<<<<<<<` and `=======` includes your local changes that haven't been pushed to the remote repository yet. The code between `=======` and `>>>>>>>` contains changes from the remote repository or another branch.
```

Comment:
Sorry, this is not local or remote. It's between the source branch and the destination branch. In your example it is b1 - source, b2 - destination

Reply:
Keep the same branches in your examples across this whole page, adds clarity